defaults/main.yml

```diff
@@ -13,3 +13,6 @@ sansible_ssmtp_restrict: yes
 sansible_ssmtp_revaliases: yes
 sansible_ssmtp_use_inventory_hostname: no
 sansible_ssmtp_user: ~
+sansible_ssmtp_hostname: ''
```

Comment:
Sorry for the very late reply @filiptepper, can you put these vars in alphabetical order then we can get thie merged.

Reply:
Done!